src/views/templates/list/VirtualMachineTemplatesList.tsx

```diff
@@ -52,7 +64,9 @@ const VirtualMachineTemplatesList: FC<VirtualMachineTemplatesListProps> = ({ nam
 
   return (
     <>
-      <ListPageHeader title={t('VirtualMachine Templates')}>
+      <ListPageHeader
+        title={isEmpty(showTitle) || showTitle ? t('VirtualMachine Templates') : null}
```

Comment:
i dont understand the condition...?

Reply:
if showTitle is undefined or is true, show the title. If it's false, don't show the title. 

I can do `showTitle === false ? null :  t('VirtualMachine Templates')` instead

Reply:
!(showTitle === false) &&  t('VirtualMachine Templates') try this